### ISSUE TITLE
Feedback: "Navigate to the previous column in a grid" (Minimal Data Grid, Test 5, Command "'Shift y' (single quick key nav on)", V26.05.21)

### ISSUE THREAD

**Issue by tactics2** [w3c/aria-at#1372]
## Description of Behavior

Shift + y is not an available command in voiceover. This should be removed from the test plan.

## Test Setup

- Test File: [test-23-navToPrevColumnGrid-voiceover_macos.collected.html](https://aria-at.netlify.app/tests/apg/minimal-data-grid/test-23-navToPrevColumnGrid-voiceover_macos.collected.html)
- AT: VoiceOver for macOS (Version 15.6.1)
- Browser: Safari (Version 26)
- Command: `'Shift+y' (single quick key nav on)`
